protected/humhub/modules/user/views/auth/login_modal.php

```diff
@@ -34,14 +34,21 @@
             <div class="tab-content">
                 <div class="tab-pane <?= (!isset($_POST['Invite'])) ? "active" : ""; ?>" id="login">
 
+                    <?php if (Yii::$app->session->hasFlash('error')): ?>
```

Comment:
In this case, would `error` being used over `warning` be necessary? Using error seems a little extreme in this case

Reply:
In the non-modal login view there is currently only an "Error" output. Adding a warning block in both Views would be too much and we try to keep View changes as little as possible (or at least to combine them with other changes).